src/controllers/user.controller.js

```diff
@@ -169,13 +169,15 @@ class userController {
    */
   static async sendResetPasswordLink(req, res) {
     const result = await UserServices.findUserByEmail(req.body.email);
-    if (result !== null) {
+    if (result !== null && result.authtype === null) {
       const token = GenerateToken({ email: req.body.email, isVerified: result.isVerified, id: result.id });
       const emailView = mailer.resetPasswordView(token, result.firstName);
       mailer.sendEmail(req.body.email, 'Reset Password', emailView);
```

Comment:
@mujohn26 what if the email has not been sent successfully? You will still send the response that says that email has been sent to their inbox.